Editor/UI/ProjectSettings/CreatorSettingProvider.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Innoactive.Creator.Core.Utils.Logging;
+using Innoactive.CreatorEditor.UI;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+internal class CreatorSettingProvider : SettingsProvider
+{
+    const string Path = "Project/Creator/Settings";
```

Comment:
```suggestion
    const string Path = "Project/Creator/Logging";
```

Would call it Logging. `Settings` sounds too generic :)

Reply:
thought about adding more stuff into the settings page later on.